saleor/graphql/order/mutations/orders.py

```diff
@@ -1,7 +1,9 @@
 import graphene
 from django.core.exceptions import ValidationError
+from prices import Money
 
 from ....account.models import User
+from ....checkout.capture_payment import capture_payments
```

Comment:
Why do you place it in the checkout module? It definitely looks like logic related to order.